src/frontend/app/features/cloud-foundry/tabs/cloud-foundry-organizations/cloud-foundry-organization-spaces/tabs/cloud-foundry-space-summary/cloud-foundry-space-summary.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="space-summary">
```

Comment:
This removes the width, height 100%  - should we also do this on the org summary for consistency?